lib/galaxy/workflow/steps.py

```diff
@@ -33,6 +33,18 @@ def order_workflow_steps(steps):
         if not step.position or "left" not in step.position or "top" not in step.position:
             position_data_available = False
     if position_data_available:
+        # find minimum left and top values to normalize position
```

Comment:
The traceback from the selenium tests looks relevant:
```
galaxy.web.framework.decorators ERROR 2022-03-31 09:38:09,716 [pN:main,p:4527,tN:WSGI_7] Uncaught exception in exposed API method:
Traceback (most recent call last):
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/web/framework/decorators.py", line 336, in decorator
    rval = func(self, trans, *args, **kwargs)
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/webapps/galaxy/api/workflows.py", line 620, in update
    workflow_update_options,
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/managers/workflows.py", line 466, in update_workflow_from_raw_description
    dry_run=dry_run,
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/managers/workflows.py", line 585, in _workflow_from_raw_description
    attach_ordered_steps(workflow, steps)
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/workflow/steps.py", line 17, in attach_ordered_steps
    ordered_steps = order_workflow_steps(steps)
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/workflow/steps.py", line 37, in order_workflow_steps
    min_left = min(step.position["left"] for step in steps)
ValueError: min() arg is an empty sequence
```

Reply:
Thanks! Weird coincidence, I just started looking at this PR again an hour ago.